modules/multiplayer/multiplayer_spawner.cpp

```diff
@@ -97,13 +97,7 @@ PackedStringArray MultiplayerSpawner::get_configuration_warnings() const {
 
 void MultiplayerSpawner::add_spawnable_scene(const String &p_path) {
 	SpawnableScene sc;
-	if (p_path.begins_with("uid://")) {
-		sc.uid = p_path;
-		sc.path = ResourceUID::uid_to_path(p_path);
-	} else {
-		sc.uid = ResourceUID::path_to_uid(p_path);
-		sc.path = p_path;
-	}
+	sc.path = ResourceUID::ensure_path(p_path);
 	if (Engine::get_singleton()->is_editor_hint()) {
 		ERR_FAIL_COND(!ResourceLoader::exists(p_path));
```

Comment:
```suggestion
	sc.path = ResourceUID::ensure_path(p_path);
	if (Engine::get_singleton()->is_editor_hint()) {
		ERR_FAIL_COND(!ResourceLoader::exists(sc.path));
```

Although the check doesn't seem to ever be executed...

Reply:
Poke @KoBeWi, this doesn't seem to be included in your later amend.